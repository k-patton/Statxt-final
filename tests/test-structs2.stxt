struct Book {
	int Id;
	string Title;
	string Author;
};

struct Chapter {
	string Title;
	int Number;
};

int main()
{
	struct Book b1;
	struct Chapter c1;

	b1.Id = 2;
	b1.Title = "Harry Potter";
	b1.Author = "JK Rowling";

	c1.Title = "Boy who lived";
	c1.Number = 1;


	print(b1.Id);
	printstr(b1.Title);
	printstr(b1.Author);
	printstr(c1.Title);
	print(c1.Number);
	return 0;
}

