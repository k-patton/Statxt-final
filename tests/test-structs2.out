2
Harry Potter
JK Rowling
Boy who lived
1
